zanata-war/src/main/java/org/zanata/servlet/ZanataDatabaseMetadataServletListener.java

```diff
@@ -58,26 +59,23 @@ public void contextInitialized(ServletContextEvent servletContextEvent) {
             String dbVersion =
                     metaData.getDatabaseProductVersion();
 
-            databaseMetaData = new ZanataDatabaseMetaData(dbProductName,
-                    dbMajorVer, dbMinorVer, dbVersion); ;
-            dbDriverMetaData = new ZanataDatabaseDriverMetadata(
-                    metaData.getDriverName(), metaData.getDriverVersion());
+            DatabaseVersionInfo databaseMetaData =
+                    new DatabaseVersionInfo(dbProductName,
+                            dbMajorVer, dbMinorVer, dbVersion);
+            DatabaseDriverVersionInfo dbDriverMetaData =
+                    new DatabaseDriverVersionInfo(
+                            metaData.getDriverName(),
+                            metaData.getDriverVersion());
 
+            log.info("===================================");
+            log.info("  Database: {}", databaseMetaData);
+            log.info("  JDBC Driver: {}", dbDriverMetaData);
```

Comment:
At this point, I'm not sure the metadata classes even deserve their own classes. But if you want to keep the classes, I would suggest making them package-private.